build-tools/build-image/cross/Dockerfile

```diff
@@ -51,14 +53,13 @@ RUN mkdir -p /usr/local/src/protobuf \
 
 # Use dynamic cgo linking for architectures other than amd64 for the server platforms
 # More info here: https://wiki.debian.org/CrossToolchains
-RUN echo "deb http://emdebian.org/tools/debian/ jessie main" > /etc/apt/sources.list.d/cgocrosscompiling.list \
-  && curl -s http://emdebian.org/tools/debian/emdebian-toolchain-archive.key | apt-key add - \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do dpkg --add-architecture ${platform}; done \
+RUN echo "deb http://cz.archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
```

Comment:
yeah, I'm not sure there's a great way to handle this without using ubuntu xenial. It seems like the s390x crossbuild just isn't available in the regular debian repos.

minor suggestion: we might want to use a generic ubuntu mirror rather than pinning to the Czech one.